app/routes/_index.tsx

```diff
@@ -35,19 +53,31 @@ export default function Index() {
             typografi={TypografiTyper.StegHeadingH1}
           />
           <Språkvelger />
+
           <VeilederHilsen
             innhold={tekster.veilederhilsenInnhold}
             hilsen={tekster.brukerHilsen}
           />
           <SamtykkePanel
+            tittel={tekster.samtykkePanelTittel}
             innhold={tekster.samtykkePanelMelding}
             samtykke={tekster.samtykkePanelSamtykke}
             feilmelding={tekster.samtykkePanelFeilmelding}
+            vedSamtykkeEndring={håndterSamtykkeEndring}
```

Comment:
```suggestion
            håndterSamtykkeEnding={håndterSamtykkeEndring}
```

Reply:
Denne funker ikke, det er allerede en egen metode i `SamtykkePanel` som heter `håndterSamtykkeEndring`. Jeg ser riktignok problemet - endret navn på den interne metoden, slik at dette funker^. Ta gjerne en titt for å se om du sier deg enig i at det gir mer mening nå 🤓